docs/docs/getting-started.md

```diff
@@ -37,13 +37,68 @@ $ browserify -t [ babelify --presets [ react ] ] main.js -o bundle.js
 To install React DOM and build your bundle with webpack:  
 
 ```sh
-$ npm install --save react react-dom babel-preset-react
+$ npm install --save react react-dom babel-preset-react babel-preset-es2015
 $ webpack
 ```
 
-> Note:
->
-> If you are using ES2015, you will want to also use the `babel-preset-es2015` package.
+A good starting webpack config for production mode would be:
+```javascript
+module.exports = {
+  entry: [
+    './client/index.jsx',
+  ],
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: 'bundle.js',
+  },
+  resolve: {
+    modulesDirectories: ['node_modules', 'client'],
+    extensions: ['.js', '.jsx'],
+  },
+  module: {
+    loaders: [{
+      test: /\.jsx?$/,
+      exclude: /node_modules/,
```

Comment:
Rather than using `exclude` it's better to use `include: path.join(__dirname__, 'src')` or equivalent because this doesn't break for npm link'd modules.